onnx/defs/tensor/defs.cc

```diff
@@ -878,7 +894,7 @@ ONNX_OPERATOR_SET_SCHEMA(
             "steps",
             "1-D tensor of slice step of corresponding axis in `axes`. "
             "Negative value means slicing backward. 'steps' cannot be 0. "
-            "Defaults to 1.",
+            "Defaults to 1s.",
```

Comment:
Thanks for the update. The CIs now all passed! 

Is this `1s` here (L897) intended or a typo?

Reply:
A slice can have more than one thing in it, so "1s" as in `(1, 1, ...)`